src/backend/distributed/citus.sql

```diff
@@ -191,6 +191,27 @@ COMMENT ON FUNCTION master_create_distributed_table(table_name regclass,
                                                     distribution_method citus.distribution_type)
     IS 'define the table distribution functions';
 
+CREATE FUNCTION master_update_shard_statistics(relation_id regclass, shard_id bigint)
+    RETURNS bigint
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$master_update_shard_statistics$$;
+COMMENT ON FUNCTION master_update_shard_statistics(relation_id regclass, shard_id bigint)
+    IS 'update shard statistics and return the updated shard size';
+
+CREATE FUNCTION is_distributed_table(relation_id regclass)
```

Comment:
Almost all of our other functions are prefixed with master or worker. Should we do the same here? Alternatively, would it be sufficient to check whether the metadata function returns an error?